api/clients/axios.ts

```diff
@@ -9,14 +9,14 @@ export class AxiosClient {
     })
   }
 
-  get<T = any, D = any, R = AxiosResponse<T, D>>(
+  get<T = any, D = any, R = AxiosResponse<T, D>>( // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
The lint warning this line gives expected is disabling it - there are a couple of lines that are similarly disabled